src/main/resources/application.properties

```diff
@@ -25,6 +25,20 @@ auth.public-key=classpath:certs/public.pem
 auth.jwt-access-expiration-minutes=60
 auth.jwt-refresh-expiration-days=7
 
+# Email config
+spring.mail.host=
+spring.mail.port=
+spring.mail.username=
+spring.mail.password=
```

Comment:
Will we need to configure this in prod? It'd be nice to have some example on the wiki if so

Reply:
Where is it? I may be legally blind